client/components/largeCarousel.jsx

```diff
@@ -1,9 +1,75 @@
 import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
 
 function LargeCarousel(props) {
   const { picture } = props;
+
+  const LargeGrid = styled.div`
+    height: 100%;
+    width: 100%;
+    display: grid;
+    grid-template-rows: 1fr 48px 1fr;
+    grid-template-columns: 60px 1fr 60px;
+  `;
+
+  const LargePicArea = styled.div`
+    grid-row-start: 1;
+    grid-row-end: 4;
+    grid-column-start: 1;
+    grid-column-end: 4;
+    display:flex;
+    width: 100%;
+    justify-content: center;
+    height: 100%
+  `;
+  const Image = styled.img`
+    max-width: 100%;
+    height: 100%;
+    border-radius: 6px;
+  `;
+  const ButtonLeft = styled.button`
+  grid-row-start: 2;
+  grid-row-end: 3;
+  grid-column-start:1;
+  grid-column-end: 2;
+  border-radius: 50%;
+  z-index: 10;
+  min-width: 48px;
+  min-height: 48px;
+  margin-left: 12px;
+  `;
+
+  const ButtonRight = styled.button`
+  grid-row-start: 2;
+  grid-row-end: 3;
+  grid-column-start: 3;
+  grid-column-end: 4;
+  border-radius: 50%;
+  z-index: 10;
+  min-width: 48px;
+  min-height: 48px;
+  margin-right:12px;
+  `;
```

Comment:
Suggestion to refactor this later to one button component and you could set the grid-column values based on props